tests/IBFE/explicit_ex4.cpp

```diff
@@ -557,7 +557,8 @@ main(int argc, char** argv)
 
         // First test for markers: add them at the start (but don't clobber
         // restart marker data)
-        if (input_db->getBoolWithDefault("test_markers", false) && time_integrator->getNumberOfMarkers() == 0)
+        if (input_db->getBoolWithDefault("test_markers", false) && time_integrator->getNumberOfMarkers() == 0 &&
+            !RestartManager::getManager()->isFromRestart())
```

Comment:
In general we try to avoid making control flow depend on whether or not we are in a restarted run - user apps should work without having to add these kinds of checks. Can we fix the bug without adding this?

Reply:
The original flags now work because `getNumberOfMarkers() != 0`--this was just an attempt to make the test more explicit.

Reply:
A check that `loop_time > 0.0` would accomplish the same thing to demonstrate that this is from a restart, or a `tbox::pout << "Initial markers added\n"` which would only show up in the `output` if no restart markers were found.